packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -92,6 +92,22 @@ const OutdentWhenBackOn1stEmptyLine: BuildInEditFeature<PluginKeyboardEvent> = {
     handleEvent: toggleListAndPreventDefault,
 };
 
+/**
+ * IndentWhenDeleteBefore1stItem edit feature, provides the ability to indent the list if user press
+ * DELETE before the first item of a list
+ */
+const IndentWhenDeleteBefore1stItem: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.DELETE],
+    shouldHandleEvent: (event, editor) => {
+        let li = editor.getElementAtCursor('LI', null /*startFrom*/, event);
```

Comment:
can we also check if this is at the end of current line, and next line is under LI? Otherwise each time we press DELETE this code will be executed, which may have impact to performance.

Reply:
Good point!! I'll fix it.